webpack/js/components.js

```diff
@@ -71,7 +71,7 @@ export const IssueCard = {
   computed: {
     dateCreated() {
       const dateCreated = new Date(this.issue.createdAt * 1000)
-      return dateCreated.toLocaleDateString()
+      return dateCreated.toISOString()
```

Comment:
I believe this method would return a string like `'2020-09-30T00:00:00.000Z'` whereas we would preferably want just the date component `'2020-09-30'`.

Reply:
hey, @dhruvkb  should I return a substring here, by formatting the isoString?

Reply:
Yeah I think that should be fine.